src/component/feature/FeatureView2/FeatureStrategies/FeatureStrategiesList/FeatureStrategiesList.styles.ts

```diff
@@ -2,7 +2,7 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles(theme => ({
     sidebar: {
-        width: '30%',
+        width: '500px',
```

Comment:
why 500px? Will that work for all devices?

Reply:
> Strategy constraints gets a bit cluttered:
> 
> ![image](https://user-images.githubusercontent.com/158948/141355140-6a639c58-2e24-426f-9814-37ba72c2c18f.png)

Yeah. We changed this end of day today and still needs some polishing. Will fix tomorrow!

Reply:
> why 500px? Will that work for all devices?

It's a way to keep it a sensible length on desktop as we now increase the width of the container when the screen is wider. Below a certain treshold we set the width of the sidebar to 100%.